ports/scintilla/portfile.cmake

```
@@ -13,10 +11,20 @@ vcpkg_extract_source_archive_ex(
   REF 4.2.3
 )
```

```suggestion
  PATCHES ${PATCHES}
)
```
You can then conditionally set `PATCHES` based on `VCPKG_CRT_LINKAGE`:
```cmake
if(VCPKG_CRT_LINKAGE STREQUAL "static")
  set(PATCHES 0001-static-build.patch)
else()
  set(PATCHES)
endif()
```

---

Thank you for the tip.

---

@ras0219-msft could you please approve the change. I would like to update Scintilla to the newest version in another PR, but this one is hanging for so long.